simulation_parameters/microbe_stage/organelles.json

```diff
@@ -1098,5 +1098,57 @@
     "RequiresNucleus": true,
     "EditorButtonGroup": "Multicellular",
     "EditorButtonOrder": 2
+  },
+  "ferroplast": {
+    "MPCost": 45,
+    "InitialComposition": {
+      "ammonia": 2,
+      "phosphates": 2
+    },
+    "Hexes": [
+      {
+        "r": 0,
+        "q": 0
+      },
+      {
+        "r": -1,
+        "q": 0
+      }
+    ],
+    "Processes": {
+      "ferrosynthesis": 1
+    },
+    "Components": {
+      "Storage": {
+        "Capacity": 1
+      }
+    },
+    "Enzymes": {
+      "rusticyanin": 1
+    },
+    "ProkaryoteChance": 0,
+    "ChanceToCreate": 0.5,
+    "Density": 1100,
+    "DisplayScene": "res://assets/models/organelles/Ferroplast.tscn",
+    "Name": "Ferroplast",
+    "ProductionColour": "#3293f7",
+    "ConsumptionColour": "#f04681",
+    "IconPath": "res://assets/textures/gui/bevel/parts/FerroplastIcon.png",
+    "UnlockConditions": [
+      {
+        "AtpProductionAbove": {
+          "Atp": 25
+        },
```

Comment:
Why is there this kind of condition? What's the reasoning why ferroplast is unlocked when you have a big cell producing a bunch of ATP?

Reply:
I just had that in there because the mitochondrion also used it. I didn't know if it was worth having there or not.

Reply:
I'd remove this unlock condition. Also with endosymbiosis merged, you should mark the ferroplast as the upgraded version of rusticyanin so that it becomes available through endosymbiosis.